cran-comments.md

```diff
@@ -13,19 +15,27 @@ There were no ERRORs or WARNINGs
 There were two NOTEs
 
 > * checking CRAN incoming feasibility ... NOTE
+> Maintainer: 'Rahul Satija <seurat@nygenome.org>'
+> 
+> New maintainer:
+>   Rahul Satija <seurat@nygenome.org>
+> Old maintainer(s):
+>   Paul Hoffman <seurat@nygenome.org>
 > Suggests or Enhances not in mainstream repositories:
 >   BPCells
 > Availability using Additional_repositories specification:
 >   BPCells   yes  https://bnprks.r-universe.dev
 
+The new maintainer is Rahul Satija, the email address has remained the same.
+
 > * checking package dependencies ... NOTE
 > Package suggested but not available for checking: 'BPCells'
 
-BPCells is hosted on R-universe and used conditionally in SeuratObject
+BPCells is hosted on R-universe and used conditionally in SeuratObject.
 
 ## Downstream dependencies
+There are 2 packages that depend on SeuratObject: Seurat, and tidyseurat; this update does not impact their functionality.
 
-The following reverse dependency is impacted by this release of SeuratObject:
+There are 9 packages that import Seurat: APackOfTheClones, bbknnR, CAMML, Platypus, scAnnotate, scaper, scCustomize, scpoisson, and Signac; this update does not impact their functionality.
 
-- tidyseurat
-  - test failures: SeuratObject changes the order of the results, but not the actual values. The authors of tidyseurat are aware of this, but the functionality of tidyseurat is not impacted
+There are 10 packages that suggest Seurat: cellpypes, CytoSimplex, RESET, rliger, scOntoMatch, SCpubr, singleCellHaystack, SpaTopic, SPECK, and VAM; this update does not impact their functionality.
```

Comment:
Are you checking packages that import/suggest Seurat or SeuratObject?

Reply:
For `SeuratObject` I ran checks manually (only things that were included with the distributed packages) - I wasn't able to install a couple of these with R 4.4 but I did my best to take a look at relevant examples/vignettes and convince myself these shouldn't be breaking changes

I inherited a script for running reverse dependency checks for `Seurat` but I may try to replicate something like what I did here

Reply:
I mention this because the `cran-comments` reference Seurat, not SeuratObject. While `cran-comments.md` is not bundled in the final tarball (it's just used to copy-and-paste into the CRAN submission form), it's still a bit of an oversight to say that these are reverse dependencies of Seurat, not SeuratObject

Reply:
Oh! That was a copy/paste error, thanks for catching it 👍 And also agreed